### ISSUE TITLE
Unable to use select.

### ISSUE THREAD

**Issue by haani104** [Dogfalo/materialize#2441]
I am having issue in select element of materializecss. I am using Angularjs + materializecss

In my jade file

    //Stylesheets
    		link(rel='stylesheet', href='https://cdnjs.cloudflare.com/ajax/libs/materialize/0.97.2/css/materialize.min.css' )

    //Vendor Scripts
    		script(src='https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js')
    		script(src='https://cdnjs.cloudflare.com/ajax/libs/materialize/0.97.2/js/materialize.min.js')

In my controller I am initializing the component as 

    $('select').material_select();

In my HTML I am using as

    <label>Materialize Select</label>
    <select>
       <option value="" disabled selected>Choose your option</option>
       <option value="1">Option 1</option>
       <option value="2">Option 2</option>
       <option value="3">Option 3</option>
    </select>

However when I try to select the options nothing happens.

below is what displayed after initialization

[![enter image description here][1]][1]

  [1]: http://i.stack.imgur.com/iw4BK.png

HTMl that gets generated looks like below

    <div class="col s4 input-field">
       <div class="select-wrapper">
          <span class="caret">▼</span><input type="text" class="select-dropdown" readonly="true" data-activates="select-options-b9fd5517-0d0d-40ff-ab29-d37fc9a8d05f" value="Choose your option">
          <select class="initialized">
             <option value="" disabled="" selected="">Choose your option</option>
             <option value="1">Option 1</option>
             <option value="2">Option 2</option>
             <option value="3">Option 3</option>
          </select>
       </div>
    </div>
As you can see ul element is not getting generated which generated after initialization.

Any clue what could be issue here.

**Comment by haani104** [comment]
Oops..
It was a version issue
I was using 0.97.2
